microshift_backup_and_restore/microshift-backup-and-restore.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="microshift-backup-and-restore"]
+= About backing up and restoring {product-title} data
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-backup-and-restore
+
+toc::[]
+
+{product-title} automatically creates a backup on every start. These automatic backups are deleted and replaced with the latest backup each time the system restarts. If you are using an RPM-OSTree system, both backup and restore are automated.
```

Comment:
> {product-title} automatically creates a backup on every start. (...) If you are using an RPM-OSTree system, both backup and restore are automated.

On non-ostree system there is no automation related to backing up or restoring the data. It's all manual.